### ISSUE TITLE
Cannot access _dbs or _nodes databases through 5984:/_node/<nodename>

### ISSUE THREAD

**Issue by wohali** [apache/couchdb#1155]
We'd like to deprecate port 5986 in CouchDB 3.0. That means everything that we currently do with port 5986 must be possible through port 5984, or other administrative means.

## Expected Behaviour
`curl http://localhost:5984/_node/_local/_dbs` should allow me access to the `_dbs` database.
`curl http://localhost:5984/_node/_local/_nodes` should allow me access to the `_nodes` database.

## Current Behaviour
With a `dev/run -n 1 --with-admin-party-please` node running in the background:

```
$ curl localhost:15984/_node/_local/_dbs
{"error":"not_found","reason":"missing"}
$ curl localhost:15984/_node/_local/_nodes
{"error":"not_found","reason":"missing"}
$
$ curl localhost:15984/_membership
{"all_nodes":["node1@127.0.0.1"],"cluster_nodes":["node1@127.0.0.1"]}
$ curl localhost:15984/_node/node1@127.0.0.1/_dbs
{"error":"not_found","reason":"missing"}
$ curl localhost:15984/_node/node1@127.0.0.1/_nodes
{"error":"not_found","reason":"missing"}
```

## Context
Prerequisite for deprecation of port 5986 (couch_httpd).

## Your Environment
* Version used: master
* Browser Name and version: curl

**Comment by wohali** [comment]
Discussion with @rnewson and @chewbranca on IRC lead to the realization that we don't, strictly speaking, need the current `localhost:5986` `/_dbs` `/_nodes` experience over on the public port. Exposing the weirdness that are these two system databases as a "standard" CouchDB database API can lead to confusion over what it really is, which is an auto-magically internally replicating un0sharded local database.

What we need to be able to do is all the things admins typically do with these databases. Here's a partial list off the top of my head:
* the equivalent of `GET _all_docs`
* the equivalent of `GET {doc}`
* the equivalent of `PUT {doc}` (for new docs, or for changed docs)
* the equivalent of SOMETHING like `_changes`:
  - to be notified when a node is added or changed
  - to be notified when a db is added, or its shard map is changed
    - can we get some/all of this from `_global_changes`?
* make sure neither `_dbs` nor `_nodes` have any document conflicts (BAD!!!) as a health check
* shard-level monkeying stuff that needs deeper thought

The ultimate end user goal is to support nice "DBA" scenarios, things like renaming a database in-place without the weight of replication.

**Comment by wohali** [comment]
Fixed in 3.0.0.
